owasp-top10-2021-apps/a2/snake-pro/app/api/routes.go

```diff
@@ -6,9 +6,12 @@ import (
 	"os"
 	"time"
 
+	"golang.org/x/crypto/bcrypt"
+
 	jwt "github.com/dgrijalva/jwt-go"
 	db "github.com/globocom/secDevLabs/owasp-top10-2021-apps/a2/snake-pro/app/db/mongo"
-	"github.com/globocom/secDevLabs/owasp-top10-2021-apps/a2/snake-pro/app/pass"
+
+	// "github.com/globocom/secDevLabs/owasp-top10-2021-apps/a2/snake-pro/app/pass"
```

Comment:
You can use this module to implement your new rules, don't need to exclude them. If you really want to remove this, don't forget to delete the file and this line.